### PR TITLE
ENH - Navbar and separate pages

### PR DESCRIPTION
**Pull request by melissaroses**
Navigation bar, content on their respective pages, changed link color to match logo, changed font.

### CODE REVIEW

File: elspHome.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Equitable Letters for Space Physics</title>
```
Comment (aburrell) [inline elspHome.html:4]:
```suggestion
        <title>ELSP - Home</title>
```

File: elspContact.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Equitable Letters for Space Physics</title>
```
Comment (aburrell) [inline elspContact.html:4]:
```suggestion
        <title>ELSP - Contact</title>
```

File: elspLetterResources.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Equitable Letters for Space Physics</title>
```
Comment (aburrell) [inline elspLetterResources.html:4]:
```suggestion
        <title>Letter Writing Resources - ELSP</title>
```

File: elspStudies.html [hunk 4]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Equitable Letters for Space Physics</title>
```
Comment (aburrell) [inline elspStudies.html:4]:
```suggestion
        <title>Gender Studies - ELSP</title>
```

**Review by aburrell — COMMENTED** [review]
Recommend changing the tab names.

File: elspBiasResources.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Equitable Letters for Space Physics</title>
```
Comment (melissaroses) [inline elspBiasResources.html:4]:
```suggestion
        <title>ELSP Addressing Bias</title>
```

**Review by aburrell — APPROVED** [review]

**Pull request merged**